components/eam/bld/configure

```diff
@@ -1455,8 +1455,8 @@ else {
 
         # MMF variance transport tracers
         if (defined $opts{'use_MMF_VT'}) {
-            $nadv += 2; # add tracers for CRM t and q variance transport
-            if ($print>=2) { print "Advected variance constituent added for MMF: 2$eol"; }
+            $nadv += 3; # add tracers for CRM variance transport if t, q, and u
+            if ($print>=2) { print "Advected variance constituent added for MMF: 3$eol"; }
```

Comment:
just a question here. What does this if statement mean? especially '>=2' part

Reply:
The "print" variable is just a flag to indicate the level of verbosity. There's a comment near the top of configure that explains the options:

# Define 3 print levels:
# 0 - only issue fatal error messages
# 1 - only informs what files are created (default)
# 2 - verbose

Reply:
thanks for clarifying that. didn't realize there's an explanation at the top of the configure file.